content/integrations/cloud_foundry.md

```diff
@@ -0,0 +1,284 @@
+
+* **Datadog Agent BOSH release** — Install the Datadog Agent on every node in your deployment to track system, network, and disk metrics. Enable any other Agent checks you wish.
+* **Datadog Firehose Nozzle** — Deploy one or more Datadog Firehose Nozzle jobs. The jobs tap into your deployment's Loggregator Firehose and send all non-container metrics to Datadog.
+* **Datadog plugin for BOSH Health Monitor** — Configure your BOSH Director's Health Monitor to send heartbeats (as metrics) and alerts (as events) from each node's BOSH Agent to Datadog.
+
+These integrations are meant for Cloud Foundry deployment administrators, not end users.
+
+# Prerequisites
+
+You need to have a working Cloud Foundry deployment and access to the BOSH Director that manages it. You also need BOSH CLI to deploy each integration. You may use either major version of the CLI—[v1](https://bosh.io/docs/bosh-cli.html) or [v2](https://bosh.io/docs/cli-v2.html#install).
+
+To configure the Datadog plugin for BOSH Health Monitor, you need access to the `state.json` (or similarly named) file that accurately reflects the current state of your BOSH Director. If you don't have such a file, you'll need to [create one](https://bosh.io/docs/cli-envs.html#deployment-state).
+
+# Install the Datadog Agent BOSH Release
+
+Datadog provides tarballs of the Datadog Agent packaged as a BOSH release. You can upload the latest release to your BOSH Director and then easily install it on every node in your deployment as an [addon](https://bosh.io/docs/runtime-config.html#addons) (i.e. the same way a Director deploys the BOSH Agent to all nodes).
+
+### Upload Datadog's release to your BOSH Director
+
+~~~
+# BOSH CLI v1
+bosh upload release 
+
+# BOSH CLI v2
+bosh upload-release 
+~~~
+
+If you'd like to create your own release, see the [Datadog Agent BOSH Release repository](https://github.com/DataDog/datadog-agent-boshrelease).
+
+### Configure the Agent as an addon in your BOSH Director
+
+Add the following to your BOSH Director's runtime configuration file (e.g. `runtime.yml`):
+
+~~~
+---
+releases:
+  - name: datadog-agent
+    version: <VERSION_YOU_UPLOADED> # specify the real version, i.e. x.y.z, not 'latest'
+
+addons:
+- name: datadog
+  jobs:
+  - name: dd-agent
+    release: datadog-agent
+  properties:
+    dd:
+      use_dogstatsd: yes
+      dogstatsd_port: 18125               # Many CF deployments have a StatsD already on port 8125
+      api_key: <YOUR_DATADOG_API_KEY>
+      tags: ["cloudfoundry_deployment_1"] # any tags you wish
+      generate_processes: true            # to enable the process check
+~~~
+
+To see which `datadog-agent` release version you uploaded earlier, run `bosh releases`.
+
+If you don't have a local copy of the runtime configuration, get it from the Director (`bosh runtime-config`) and add the above to it. If the Director's runtime configuration is empty, add the above to a new `runtime.yml`.
```

Comment:
If you don't have a local copy of the runtime configuration, it can be retrieved from the Director (`bosh runtime-config`). Once retrieved add the above addon configuration to it with your account credentials. If the Director's runtime configuration is empty, create a new `runtime.yml` and add the Datadog configuration there.